code/game/objects/structures/lattice.dm

```diff
@@ -86,13 +86,13 @@
 
 /obj/structure/lattice/catwalk/Move()
 	var/turf/T = loc
-	for(var/obj/structure/cable/C in T)
+	for(var/obj/structure/cable/C in T.nullspaced_contents)
```

Comment:
I am once again asking if nullspaced_content is something an object is always in, or only sometimes